Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1109,7 +1109,9 @@ class BrowserViewController: UIViewController {
     /// Updates the URL bar security, text and button states.
     fileprivate func updateURLBar() {
         guard let tab = tabManager.selectedTab else { return }
-        if let url = tab.url, url.isLocal {
+        if let url = tab.url, !url.isLocal {
+            self.rewards?.ledger.fetchPublisherActivity(from: url, faviconURL: nil, publisherBlob: nil, tabId: UInt64(tab.rewardsId))
```

Comment:
This logic is still different. I think this is what you're looking for:

```swift
if let url = tab.url {
  if url.isLocal {
    self.topToolbar.locationView.rewardsButton.isVerified = false
  } else {
    self.rewards?.ledger.fetchPublisherActivity(from: url, faviconURL: nil, publisherBlob: nil, tabId: UInt64(tab.rewardsId))
  }
}
```

Also:
1. Is it possible to get the faviconURL here? 
2. Can you double check that `twitch.tv` still shows publisher names in the panel when you switch pages? (As it uses publisher blob)

Reply:
For the case if its local url or the url is nil the isVerified should be set false. In the other case it should fetch. 
The `if url.isLocal {
    self.topToolbar.locationView.rewardsButton.isVerified = false
  }` is also my code from a previous pr.

In your recommended code the verified sign will still be there if the url is set to nil

Reply:
> In your recommended code the verified sign will still be there if the url is set to nil

This is how the logic is _now_, before this change, based on how the code is written. Was the change on purpose you mean then? To make it go away if the URL is nil?

Reply:
Yeah, I realised this case when working on this pr. It should be set to false if the url is nil too. Thus corrected it in this pr. 

For twitch.tv we definitely need the blob, looking at how to fix for it too. Thanks for lmk